src/components/book-preview-list.ts

```diff
@@ -2,15 +2,21 @@ import { Component, Input } from '@angular/core';
 
 import { BookPreviewComponent, BookInput } from './book-preview';
 
-
 export type BooksInput = BookInput[];
 
 @Component({
   selector: 'book-preview-list',
   directives: [ BookPreviewComponent ],
   template: `
-    <book-preview *ngFor="let book of books" [book]="book"></book-preview>
-  `
+    <book-preview [book]="book" *ngFor="let book of books"></book-preview>
```

Comment:
Normally this wouldn't be an issue, but since this is an example app can you change the order here?

```html
*ngFor="let book of books" [book]="book"
```